src/Features/LanguageServer/Protocol/Handler/Diagnostics/IRoslynRequestHandler.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommonLanguageServerProtocol.Framework;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler
+{
+    internal interface IRoslynNotificationHandler : ILspService, INotificationHandler<RequestContext>
```

Comment:
What is the name `Roslyn` trying to convey in these types?

Reply:
I guess my thought is that if this is a NotificationHandler for a LspService, would `ILspNotificationHandler` or `ILspServiceNotificationHandler` be more meaningful.